Bulldozer/Utility/AddMethods.cs

```diff
@@ -1944,7 +1954,7 @@ public static string StripSuffix( string textValue, int? campusId )
                         ApprovedByPersonAliasId = approvedByAliasId,
                         CreatedByPersonAliasId = createdByAliasId,
                         RequestedByPersonAliasId = requestedByAliasId,
-                        ForeignKey = foreignKey,
+                        ForeignKey = string.Format( "{0}{1}", foreignKeyPrefix, foreignKey ),
```

Comment:
Follow typical format. Why append ^ before, but not here. Match usage of line 2371